GraphNeuralNetworks/docs/src_tutorials/introductory_tutorials/traffic_prediction.jl

```diff
@@ -0,0 +1,134 @@
+
+dataset_metrla = METRLA(; num_timesteps = 3)
+# 
+g = dataset_metrla[1]
+
+
+# `edge_data` contains the weights of the edges of the graph and
+# `node_data` contains a node feature vector and a target vector. The latter vectors contain batches of dimension `num_timesteps`, which means that they contain vectors with the node features and targets of `num_timesteps` time steps. Two consecutive batches are shifted by one-time step.
+# The node features are the traffic speed of the sensors and the time of the day, and the targets are the traffic speed of the sensors in the next time step.
+# Let's see some examples:
+
+features = map(x -> permutedims(x,(1,3,2)), g.node_data.features)
+
+size(features[1])
+
+# The first dimension correspond to the two features (first line the speed value and the second line the time of the day), the second to the number of timestep `num_timesteps` and the third to the nodes.
+
+targets = map(x -> permutedims(x,(1,3,2)), g.node_data.targets)
+
+size(targets[1])
+
+# In the case of the targets the first dimension is 1 because they store just the speed value.
+
+features[1][:,:,1]
+
+# 
+features[2][:,:,1]
+
+#
+targets[1][:,:,1]
+
+# 
+function plot_data(data,sensor)
+	p = plot(legend=false, xlabel="Time (h)", ylabel="Normalized speed")
+	plotdata = []
+	for i in 1:3:length(data)
+		push!(plotdata,data[i][1,:,sensor])
+	end
+	plotdata = reduce(vcat,plotdata)
+	plot!(p, collect(1:length(data)), plotdata, color = :green, xticks =([i for i in 0:50:250], ["$(i)" for i in 0:4:24]))
+	return p
+end
+
+plot_data(features[1:288],1)
+
+# Now let's construct the static graph, the `train_loader` and `data_loader`.
+
+graph = GNNGraph(g.edge_index; edata = g.edge_data, g.num_nodes);
+
+train_loader = zip(features[1:200], targets[1:200]);
+test_loader = zip(features[2001:2288], targets[2001:2288]);
+
+# ## Model: T-GCN
+
+# We use the T-GCN model from the paper [T-GCN: A Temporal Graph Convolutional Network for Traffic Prediction] (https://arxiv.org/pdf/1811.05320.pdf), which consists of a graph convolutional network (GCN) and a gated recurrent unit (GRU). The GCN is used to capture spatial features from the graph, and the GRU is used to capture temporal features from the feature time series.
+
+model = GNNChain(TGCN(2 => 100; add_self_loops = false), Dense(100, 1))
+
+# ![](https://www.researchgate.net/profile/Haifeng-Li-3/publication/335353434/figure/fig4/AS:851870352437249@1580113127759/The-architecture-of-the-Gated-Recurrent-Unit-model.jpg)
+
```

Comment:
here it would be useful to show the output of the model and how it is interpreted as a prediction